app/assets/javascripts/components/course/course.jsx

```diff
@@ -127,24 +127,55 @@ export const Course = createReactClass({
         <div className="course_main container">
           <Confirm />
           {enrollCard}
-          <Switch>
-            <Route exact path="/courses/:course_school/:course_title" render={() => <OverviewHandler {...courseProps} />} />
-            <Route exact path="/courses/:course_school/:course_title/home" render={() => <OverviewHandler {...courseProps} />} />
+          <Routes>
+            <Route
```

Comment:
I think this will be easier to read if each Route is on a single line, like before.